src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -748,6 +748,18 @@ bool CommandRequestImpl::PrepareResultForMobileResponse(
   return result;
 }
 
+bool CommandRequestImpl::PrepareResultForMobileResponse(
+    ResponseInfo& out_first,
+    ResponseInfo& out_second,
+    ResponseInfo& out_third) const {
+  SDL_LOG_AUTO_TRACE();
+  bool result = CheckResult(out_first, out_second) ||
+                CheckResult(out_second, out_first) ||
+                CheckResult(out_third, out_second) ||
+                CheckResult(out_second, out_third);
```

Comment:
@RomanReznichenkoLuxoft as this is OR it is enough to write like this:
```suggestion
  bool result = CheckResult(out_first, out_second) ||
                CheckResult(out_second, out_third);
```

Reply:
done